Makefile

```diff
@@ -105,6 +105,7 @@ build-$(3): $(3)/Gemfile $(3)/Dockerfile
 		--label metanorma-container-version=$(1) \
 		--label metanorma-container-commit=$(CONTAINER_COMMIT) \
 		--label metanorma-container-commit-branch=$(CONTAINER_BRANCH) \
+		--build-arg BUNDLE_RUBYGEMS__PKG__GITHUB__COM \
```

Comment:
@CAMOBAP don't you need `=$(BUNDLE_RUBYGEMS__PKG__GITHUB__COM)` here to propagate the environment variable? And have to provide the environment variable in the GHA `make` step?

Reply:
It's absolutely legal to pass it without value https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg

but because I haven't other idea I'm already testing this

Reply:
Doesn't work also, I see that ubuntu querying https://rubygems.pkg.github.com/metanorma/versions looks like issue with ARG/ENV commands in docker file by itself testing right now...